ethicalads-theme/templates/ea/publishers-calculator.html

```diff
@@ -0,0 +1,112 @@
+{% extends "page.html" %}
+
+
+{% block content %}
+<section class="container py-5 mb-10">
+  <article class="page">
+    <h1 class="font-weight-light display-3">{{ page.title }}</h1>
+
+    <p>There's a few important variables that determine how much you'll make with EthicalAds. We built this calculator so you know what kind of a monthly payout to expect.</p>
+
+<div id="publisher-calculator" class="card card-border border-primary shadow-light-lg">
+  <div class="card-body">
+    <div class="row justify-content-center">
+      <div class="col-md-8 col-xs-12">
+
+        <form>
+
+          <div class="form-group mb-8">
+            <h3>Monthly site pageviews</h3>
+            <p class="small">How many pageviews your site gets is the most important input into how much you'll earn with us. Use "unique pageviews" if that is broken out in your analytics.</p>
+
+            <div class="d-print-none">
+              <div class="input-group input-group-sm mx-auto w-50 w-lg-25">
+                <input type="number" class="form-control" data-bind="value: pageviews" min="20000" max="5000000" step="10000" name="pageviews-precise" id="pageviews-precise">
+              </div>
+
+              <input type="range" class="form-control-range" min="20000" max="5000000" step="10000" data-bind="value: pageviews" name="pageviews" id="pageviews">
+              <div class="d-flex">
+                <div class="small flex-fill">20K</div>
+                <div class="small flex-fill text-right">5M</div>
+              </div>
+            </div>
+          </div>
+
+          <div class="form-group mb-8">
+            <h3>Pageview estimate source</h3>
+            <p class="small">Where did you get the figure for your monthly pageviews? Different ways of counting pageviews more or less accurately reflect how many pageviews will result in a billable ad impression (see <a href="/publishers/faq/#analytics-shows-more-pageviews-than-youre-showing-ad-views-why">why that is</a>).</p>
```

Comment:
```suggestion
            <p class="small">Where did you get the figure for your monthly pageviews? Different tools will vary in how accurately they are able to count billable pageviews (see <a href="/publishers/faq/#analytics-shows-more-pageviews-than-youre-showing-ad-views-why">why that is</a>).</p>
```

Original was a bit hard to understand, I think this is a bit better, but still don't love it.